assets/js/weforms.js

```diff
@@ -3632,6 +3632,17 @@ if (window.Sweetalert2) window.sweetAlert = window.swal = window.Sweetalert2;
 	});
 
 	$.fn.extend({
+    // /**
+		// * Custom jQuery serialize wrapper.
+		// *
+		// * When WordPress 5.6 increased the jQuery version to 3.5.1, the serialize function changed. Instead of
+		// * sending spaces as "+", they are sent as "%20". This wrapper is for backwards compatibility.
+		// *
+		// * @since ###
+		// */
+		// weSerialize: function() {
+		// 	return $( this ).serialize().replaceAll( '%20', '+' );
+		// },

```

Comment:
Should we remove this?

Reply:
@bwmarkle Uncommented, this will need to stay since the weforms.js is used for weform.min.js so we will need it for this release, but will be automaticly when we create the one script for the front and backend.